llvm/test/Transforms/Inline/PowerPC/inline-target-features.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: opt < %s -mtriple=powerpc64le-unknown-linux-gnu -S -passes=inline | FileCheck %s
```

Comment:
nit: since the change is not PPC Linux specific, maybe we can add RUN lines for AIX as well?